README.md

```diff
@@ -22,8 +22,7 @@ This library uses semantic versioning and follows Okta's [Library Version Policy
 
 | Version | Status                           |
 | ------- | -------------------------------- |
-| 0.x     | :warning: Beta Release (Retired) |
-| 1.x     | :heavy_check_mark: Release       |
+| 0.x     | :warning: Beta Release           |
```

Comment:
Should be `:heavy_check_mark: Beta Release`

Reply:
Fixed